CMakeLists.txt

```diff
@@ -171,8 +171,16 @@ if(USE_CUDA)
     set(CUDA_ARCHS "6.0" "6.1" "6.2" "7.0")
     if(CUDA_VERSION VERSION_GREATER_EQUAL "10.0")
         list(APPEND CUDA_ARCHS "7.5")
+    elseif(CUDA_VERSION VERSION_GREATER_EQUAL "11.0")
+        list(APPEND CUDA_ARCHS "8.0")
```

Comment:
Could you please extract this part into a separate PR? This is not related to fixing CMake warnings.
Also, in new PR `8.6` can be appended, right?

Reply:
You are right. I will remove it in this PR

Reply:
Thank you! Please feel free to provide a PR with new architectures added.